scripts/generate_man.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import os.path
+
+from build_manpages.build_manpage import ManPageWriter
+
+from pipx.main import get_command_parser
+
+
+class FakeSetuptoolsDistribution:
+    """Implement the setuptools Distribution() getters"""
+
+    def get_name(self):
+        return "pipx"
+
+    def get_author(self):
+        return "Chad Smith and contributors"
+
+    def get_author_email(self):
+        return "grassfedcode@gmail.com"
+
+    def get_url(self):
+        return "https://pypa.github.io/pipx/"
```

Comment:
Instead of hardcoding these, I also played with extracting them from importlib_metadata, but that can only be done when it's installed, so this simpler approach seemed the right way to go.

Reply:
Actually, there is a lower level API that probably makes more sense to use, brb...